djangocms_bootstrap4/contrib/bootstrap4_tabs/constants.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.utils.translation import ugettext_lazy as _
+
+
+TAB_TYPE_CHOICES = (
+    ('nav-tabs', _('Tabs')),
+    ('nav-pills', _('Pills')),
+)
+
+TAB_ALIGNMENT_CHOICES = (
+    ('nav-fill', _('Fill')),
+    ('nav-justified', _('Justified')),
+    ('justify-content-start', _('Justify start')),
+    ('justify-content-center', _('Justify center')),
+    ('justify-content-end', _('Justify end')),
+    ('flex-column', _('Column')),
+)
+
+TAB_EFFECT_CHOICES = (
+    ('fade', _('Fade')),
```

Comment:
would be awesome to have all of this actually configurable

Reply:
Will think about this once I'm doing cleanup :)